tests/NetworkTest.py

```diff
@@ -32,15 +32,16 @@ def test_aneuron(self):
         """
         self.assertTrue(isinstance(self.net.aneuron('AVAL'),PyOpenWorm.Neuron))
 
+    @unittest.expectedFailure
```

Comment:
Great progress!  @signija do you have the info you need to remove the expected failure now before making the final merge?